velox/connectors/hive/HiveConnectorSplit.h

```diff
@@ -22,11 +22,22 @@
 
 namespace facebook::velox::connector::hive {
 
+struct FileProperties {
+  int64_t fileSize{-1};
+  int64_t modificationTime{-1};
+
+  FileProperties() = default;
+
+  FileProperties(int64_t _fileSize, int64_t _modificationTime)
+      : fileSize(_fileSize), modificationTime(_modificationTime) {}
+};
+
 struct HiveConnectorSplit : public connector::ConnectorSplit {
   const std::string filePath;
   dwio::common::FileFormat fileFormat;
   const uint64_t start;
   const uint64_t length;
+  FileProperties properties;
```

Comment:
Is this used?

Reply:
This is used in `SplitReader::createReader()`

Reply:
This is used in `SplitReader::createReader()`